typed-racket-doc/info.rkt

```diff
@@ -14,7 +15,7 @@
                      "pict-lib"
                      ("typed-racket-lib" #:version "1.10")
                      "typed-racket-compatibility"
-                     "typed-racket-more"
+                     ("typed-racket-more" #:version 1.10)
```

Comment:
I think this is the issue with the build (the version should be a string, so `("typed-racket-more" #:version "1.10"). I'll push a patch shortly.